routing_common/transit_serdes.hpp

```diff
@@ -24,6 +26,12 @@ namespace routing
 {
 namespace transit
 {
+// Note. For the time being double at transit section is used only for saving weight of edges (in seconds).
+// Let us assume that it takes less than 10^7 seconds (115 days) to get from one station to a neighboring one.
+double constexpr kMinDoubleAtTransit = kInvalidWeight;
+double constexpr kMaxDoubleAtTransit = 10000000.0;
+uint32_t constexpr kDoubleCoordBits = 31;
```

Comment:
Why do you use the word Coord? And why 31 bit? Why not to use all 32 bits?

Reply:
So, you encode double to uint32_t, but use 31 bit instead of 32. Then, you write this uint32_t as is (not as varint), therefore in the section it will occupy 32 bits, where 1 bit is unused. Why?

Reply:
> Why do you use the word Coord?

Fixed. kDoubleBits.

> And why 31 bit? Why not to use all 32 bits?

Pls have a look at the method:
```c++
uint32_t DoubleToUint32(double x, double min, double max, uint32_t coordBits)
{
  x = my::clamp(x, min, max);
  return static_cast<uint32_t>(0.5 + (x - min) / (max - min) * ((1 << coordBits) - 1));
}
```
`1 << coordBits` is (signed) int. If to write `auto k = 1 << coordBits;` will get a warning: warning: overflow in expression; result is 2147483647 with type 'int' [-Winteger-overflow]

Reply:
1<<32 = 2^32 while max for uint32_t is 2^32 - 1. Is not it so?

Reply:
So as far as I see I cann't use 32 bits at this method.

Reply:
Feel free to transform this to: ```static_cast<uint64_t>(1) << coordBits```. This should help. And ASSERT-ion that 1 <= coordBits <= 32.

Reply:
Also, you may reuse the method GetFullMask() from bits, instead of this (1 << n) - 1.

Reply:
Thanks. I'd prefer to use GetFullMask and assertion.

Reply:
So yes. It helps. 
But I guess I understand why POINT_COORD_BITS is define as 30 :)